core/deployment/src/main/java/io/quarkus/deployment/configuration/ConfigDefinition.java

```diff
@@ -71,6 +71,10 @@
             .asList(QUARKUS_NAMESPACE + ".live-reload.password", QUARKUS_NAMESPACE + ".live-reload.url",
                     QUARKUS_NAMESPACE + ".debug.generated-classes-dir", QUARKUS_NAMESPACE + ".debug.reflection",
                     QUARKUS_NAMESPACE + ".build.skip",
+                    QUARKUS_NAMESPACE + ".platform.group-id",
+                    QUARKUS_NAMESPACE + ".platform.artifact-id",
+                    QUARKUS_NAMESPACE + ".platform.version",
+                    QUARKUS_NAMESPACE + "-plugin.version",
```

Comment:
hmm - when seeing it here and in the lower pom.xml's then the `quarkus-plugin` does feel out of place as one would expect `quarkus.plugin`. 

should we make it `quarkus.plugin.version` just to not trigger other peoples OCD ? :)

Reply:
okey - just did a quick search on github and "-plugin.version" seems waay more prominent (millions vs thousands) than ".plugin.version" so lets just keep it as is.